includes/reader-revenue/class-woocommerce-connection.php

```diff
@@ -1010,6 +1011,18 @@ public static function stripe_gateway_payment_request_data( $post_data, $order )
 		}
 		return $post_data;
 	}
+
+	/**
+	 * Force allow switching the subscription amount.
+	 *
+	 * @param bool $can_switch Whether the subscription amount can be switched.
+	 */
+	public static function force_allow_switching_subscription_amount( $can_switch ) {
+		if ( defined( 'NEWSPACK_PREVENT_FORCE_ALLOW_SWITCHING' ) && NEWSPACK_PREVENT_FORCE_ALLOW_SWITCHING ) {
```

Comment:
The name of the constant is very confusing to me. Is it 
* preventing the site from allowing switch
* Forcing the site to allow the switch
* Forcing the site to prevent allowing the switch
:)

Reply:
That's right :D WDYT about `NEWSPACK_PREVENT_WC_SUBS_ALLOW_SWITCHING_OVERRIDE`?